### PR TITLE
cleanup: remove import alias for fence library along with other cleanup

### PR DESCRIPTION
**Pull request by humblec**
cleanup: remove import alias for fence library along with other cleanup
    
this commit remove unneeded import alias of fence library from the
network_fence test. also this commit correct some typos in the source
code comments.

   
 Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: e2e/cephfs.go [hunk 1]
```diff
@@ -296,7 +296,7 @@ var _ = Describe("cephfs", func() {
 				}
 			})
 
-			By("checking nodeplugin deamonset pods are running", func() {
+			By("checking nodeplugin daemonset pods are running", func() {
```
Comment (nixpanic) [inline e2e/cephfs.go:299]: this has nothing to do with the PR, nor with the commit?

**Review by nixpanic — CHANGES_REQUESTED** [review]

File: e2e/cephfs.go (hunk 1 above)
Comment (humblec) [reply e2e/cephfs.go:299]: @nixpanic mentioned or done in this commit as part of cleanup mentioned in the same ?

Comment (humblec) [reply e2e/cephfs.go:299]: if you are mentioning it becuase of "cephfs", sure, let me split these commits.

Comment (humblec) [reply e2e/cephfs.go:299]: regardless, rearranged the commits a bit.. ptal.. thanks @nixpanic

File: deploy/rbd/kubernetes/csi-provisioner-rbac.yaml [hunk 2]
```diff
@@ -86,9 +86,6 @@ metadata:
   namespace: default
   name: rbd-external-provisioner-cfg
 rules:
-  - apiGroups: [""]
```
Comment (nixpanic) [inline deploy/rbd/kubernetes/csi-provisioner-rbac.yaml]: rbd can use a configmap for KMS settings?

File: e2e/pvc.go [hunk 3]
```diff
@@ -339,7 +339,7 @@ func getMetricsForPVC(f *framework.Framework, pvc *v1.PersistentVolumeClaim, t i
 			return false, nil
 		}
 		if stdOut == "" {
-			e2elog.Logf("no metrics received from kublet on IP %s", kubelet)
+			e2elog.Logf("no metrics received from kubelet on IP %s", kubelet)
```
Comment (nixpanic) [inline e2e/pvc.go:342]: lots of typo fixes in the wrong commit :sob:

File: internal/rbd/rbd_util_test.go [hunk 4]
```diff
@@ -297,7 +297,7 @@ func TestStrategicActionOnLogFile(t *testing.T) {
 			var err error
 			switch tt.args.logStrategy {
 			case "compress":
-				newExt := strings.Replace(tt.args.logFile, ".log", ".gz", -1)
+				newExt := strings.ReplaceAll(tt.args.logFile, ".log", ".gz")
```
Comment (nixpanic) [inline internal/rbd/rbd_util_test.go]: why?

**Review by nixpanic — CHANGES_REQUESTED** [review]

File: deploy/rbd/kubernetes/csi-provisioner-rbac.yaml (hunk 2 above)
Comment (humblec) [reply deploy/rbd/kubernetes/csi-provisioner-rbac.yaml]: hmmm.. let me revisit this part then :)

File: e2e/pvc.go (hunk 3 above)
Comment (humblec) [reply e2e/pvc.go:342]: yeah.. splitting these into different commits looked ugly :) , so consolidated in one commit :)

File: internal/rbd/rbd_util_test.go (hunk 4 above)
Comment (humblec) [reply internal/rbd/rbd_util_test.go]: reverted..

File: deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml [hunk 5]
```diff
@@ -77,10 +77,6 @@ metadata:
   namespace: default
   name: cephfs-external-provisioner-cfg
 rules:
-  # remove this once we stop supporting v1.0.0
-  - apiGroups: [""]
-    resources: ["configmaps"]
-    verbs: ["get", "list", "create", "delete"]
```
Comment (Madhu-1) [inline deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml]: same changes are required in helm charts and for the rbd.

**Review by Madhu-1 — CHANGES_REQUESTED** [review]
Good to have separate PRs for typos and fixing the RBAC. helps in reviewing the code.

**Comment by Madhu-1** [comment]
@humblec as you are already working on RBAC cleanup can you please also take care of https://github.com/ceph/ceph-csi/issues/3018?

File: deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml (hunk 5 above)
Comment (humblec) [reply deploy/cephfs/kubernetes/csi-provisioner-rbac.yaml]: I am keeping this PR for just cleanup commit and will put RBAC one different.. @Madhu-1

**Comment by humblec** [comment]
> @humblec as you are already working on RBAC cleanup can you please also take care of #3018?

yeah. I am splitting this PR , so should be able to consolidate there in the new one.

**Review by Madhu-1 — DISMISSED** [review]

**Comment by Madhu-1** [comment]
> > @humblec as you are already working on RBAC cleanup can you please also take care of #3018?
> 
> yeah. I am splitting this PR , so should be able to consolidate there in the new one.

Thanks @humblec assigned #3018 to you

**Review by nixpanic — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Comment by Rakshith-R** [comment]
@mergifyio rebase

**Pull request merged**